de-lib/src/main/java/org/iplantc/de/collaborators/client/presenter/ManageCollaboratorsPresenter.java

```diff
@@ -110,6 +115,26 @@ public void onFailure(Throwable caught) {
 
     }
 
+    void updateListView() {
+        String searchTerm = "*";
```

Comment:
We really should be converting our "search" endpoints like this into "list" endpoints with an optional `search` query param (see cyverse-de/apps#47 for example).